python/cuml/linear_model/logistic_regression.pyx

```diff
@@ -423,36 +475,44 @@ class LogisticRegression(Base):
         return accuracy_score(y, self.predict(X), handle=self.handle)
 
     def get_param_names(self):
-        return ["C", "penalty", "tol", "fit_intercept", "max_iter",
-                "linesearch_max_iter", "l1_ratio", "solver"]
+        return [
+            "C",
+            "penalty",
+            "tol",
+            "fit_intercept",
+            "max_iter",
+            "linesearch_max_iter",
+            "l1_ratio",
+            "solver",
+        ]
 
     def __getstate__(self):
         state = self.__dict__.copy()
         # Remove the unpicklable handle.
-        if 'handle' in state:
-            del state['handle']
+        if "handle" in state:
+            del state["handle"]
 
-        if 'coef_' in state:
-            del state['coef_']
-        if 'intercept_' in state:
-            del state['intercept_']
+        if "coef_" in state:
+            del state["coef_"]
+        if "intercept_" in state:
+            del state["intercept_"]
         return state
 
     def __setstate__(self, state):
         super(LogisticRegression, self).__init__(handle=None,
-                                                 verbose=state['verbose'])
+                                                 verbose=state["verbose"])
 
-        if 'qn' in state:
-            qn = state['qn']
+        if "qn" in state:
+            qn = state["qn"]
             if qn.coef_ is not None:
                 if qn.fit_intercept:
-                    state['coef_'] = qn.coef_[0:-1]
-                    state['intercept_'] = qn.coef_[-1]
+                    state["coef_"] = qn.coef_[0:-1]
+                    state["intercept_"] = qn.coef_[-1]
                 else:
-                    state['coef_'] = qn.coef_
+                    state["coef_"] = qn.coef_
                     n_classes = qn.coef_.shape[1]
-                    state['intercept_'] = rmm.to_device(np.zeros(
-                        n_classes,
-                        dtype=qn.coef_.dtype))
+                    state["intercept_"] = rmm.to_device(
+                        np.zeros(n_classes, dtype=qn.coef_.dtype)
+                    )
```

Comment:
Wouldn't this be better as CumlArray.zeros?

Reply:
I think so, yes. This would be a change unrelated to the PR (this code is only showing up because I ran `black`), but I'm happy to update it. Most of the other estimators are standardized on CumlArray.zeros for output

Reply:
Resolved by https://github.com/rapidsai/cuml/pull/2403/commits/e824ef4c2d8da3b017ec59b8ef4bbe2006c2e91c